src/main/java/Nehsik.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        displayGreetings();
+        
+        Scanner in = new Scanner(System.in);
+        Task[] taskList = new Task[MAX_TASKS];
+        int taskIndex = 0;
+
+        while (true) {
```

Comment:
Very good and clear way to handle input, so much more concise as compared to what I did, thank you for the inspiration!